client/ayon_deadline/plugins/publish/houdini/submit_houdini_cache_deadline.py

```diff
@@ -78,8 +78,9 @@ def get_job_info(self):
         job_info.BatchName = batch_name
         job_info.Plugin = instance.data["plugin"]
         job_info.UserName = context.data.get("deadlineUser", getpass.getuser())
-        rop_node = self.get_rop_node(instance)
-        if rop_node.type().name() != "alembic":
+        frames = instance.data.get("frames", "")
```

Comment:
> what's the reason why we can't just ALWAYS pass `frameStart` and `frameEnd`. Shouldn't those just always be passed correctly anyway? Or what are we trying to avoid here?

we are avoiding adding a value to `job_info.Frames` when alembic node has is outputting one file we won't like to add frames.   or at least this is what I've found out from experimenting.

Reply:
Yeah, number of frames vs chunk size is an issue. 
But, will users be mad if the chunk size magically changed during publishing?
![image](https://github.com/user-attachments/assets/11877c24-5494-4bdc-b5ac-2b72a1c8225b)

Reply:
It's done in Nuke as well - if single video file is written, we magically increase the chunk size to a massive number to avoid multiple concurrent machines writing into the file. I haven't seen any way in Deadline to do that differently :(

I think it's [this one](https://github.com/ynput/ayon-deadline/blob/develop/client/ayon_deadline/plugins/publish/nuke/submit_nuke_deadline.py#L355) for Nuke submissions.

Reply:
Thank you 
Here it's https://github.com/ynput/ayon-deadline/pull/62/commits/790c36f20d8f582df323b503f55987777b9be685

Reply:
Aah - I forgot one caveat actually. I believe this `chunkSize` doesn't work with by `byFrameStep` :( because as soon as you skip frames, then Deadline will never chunk them.

Reply:
Yeah, right.
I'm amazed that the job didn't error
![image](https://github.com/user-attachments/assets/c07dd259-52aa-4f9a-ba0e-e7ba391467dc)

Reply:
Tagging @moonyuet for visibility.

Reply:
> Yeah, right. I'm amazed that the job didn't error _Edit: The files override each other, and I've only the last frame._
> 
> ![image](https://private-user-images.githubusercontent.com/20871534/388702883-c07dd259-52aa-4f9a-ba0e-e7ba391467dc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIyNjg5MjksIm5iZiI6MTczMjI2ODYyOSwicGF0aCI6Ii8yMDg3MTUzNC8zODg3MDI4ODMtYzA3ZGQyNTktNTJhYS00ZjlhLWJhMGUtZTdiYTM5MTQ2N2RjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTIyVDA5NDM0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY0M2U3N2IyNzMxMjMzNTAzNThkMDUyN2JiNWJhYzE0ZjhhZjVlMDdmZWQwNjQyMmYyYmM3MGMyYzIxMTRkOGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.JLUfd8MJQYHtxMg-uoV6ymFeHlsrI8LxYOiiJBK95e4)

we can do the same way as we did in the Nuke, i.e. adding massive number of chucksize to avoid this issue happened.

Reply:
have you tried to manually submit Houdini Scene with massive number of chuck size and also with the frame step to see if the issue still persists? If yes, then it shouldn't be the just the issue from our side.

Reply:
yes, tested that in my screenshot above where I used the deadline OTL.
So, the issue is not from our side. cool!